kolibri/deployment/default/cache.py

```diff
@@ -0,0 +1,47 @@
+import sys
+
+from kolibri.utils.conf import OPTIONS
+
+cache_options = OPTIONS["Cache"]
+
+# Default to LocMemCache, as it has the simplest configuration
+default_cache = {
+    "BACKEND": "django.core.cache.backends.locmem.LocMemCache",
+    # Default time out of each cache key
+    "TIMEOUT": cache_options["CACHE_TIMEOUT"],
+}
+
+built_files_prefix = "built_files"
+
+built_files_cache = {
+    "BACKEND": "django.core.cache.backends.locmem.LocMemCache",
+    # Default time out of each cache key
+    "TIMEOUT": cache_options["CACHE_TIMEOUT"],
+}
+
+
+if cache_options["CACHE_BACKEND"] == "redis":
+    if sys.version_info.major == 3 and sys.version_info.minor < 5:
+        raise RuntimeError(
+            "Attempted to use redis cache backend with Python 3.4, please use Python 2.7 or 3.5+"
+        )
+    base_cache = {
+        "BACKEND": "redis_cache.RedisCache",
+        "LOCATION": cache_options["CACHE_LOCATION"],
+        # Default time out of each cache key
+        "TIMEOUT": cache_options["CACHE_TIMEOUT"],
+        "OPTIONS": {"PASSWORD": cache_options["CACHE_PASSWORD"]},
+    }
+    default_cache = dict(base_cache)
```

Comment:
`base_cache` already seems like a dict! What's the reason for a casting it again?

Reply:
To make a copy, so that I can edit it - although, I am now realizing that because it has a subdict, I need to do a deep copy instead. Good catch!